packages/mdc-select/mdc-select.scss

```diff
@@ -261,6 +261,14 @@ $mdc-select-menu-transition: transform 180ms $mdc-animation-standard-curve-timin
         @include mdc-theme-prop(color, text-primary-on-dark);
       }
     }
+
+    // Styles to override mdc-ripple-radius from mdc-list, which causes display glitches in Chrome
+    @include mdc-ripple-radius(50%);
+
+    &::before,
+    &::after {
+      border-radius: 0;
```

Comment:
Is this needed? When I comment it out, the display glitch still seems to be fixed.

Reply:
Never mind, I'm dumb. This is definitely needed to fix the ripple on press.